### PR TITLE
Update consistency for requirements vs specifications

### LINKED ISSUE #20: update references to "requirements" vs "specifications"
resolve inconsistencies around language: requirements vs. specifications

### PR DESCRIPTION
**Pull request by elimillera**
This closes #20 

'grep' says I got it all but its not impossible I'm missing something.

Other small changes:
I moved the `@importFrom usethis edit_file` to the zzz.R. From what I've seen its generally best practice to put onLoads, imports, and main package man page in a file called zzz.R or <packageName>-package.R, with the latter being more popular these days.

The man pages for `vt_use_*` all use the same man page so I removed redundant parts and made it more general.

### CODE REVIEW

File: R/zzz.R [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#' @importFrom usethis edit_file
```
Comment (thebioengineer) [inline R/zzz.R:1]: Interesting. I've not seen folks moving dependencies away from the functions they are called in. To me that has the potential of accidentally removing a dependency (import) when you think you no longer need it and cluttering up another location 🤔

Comment (thebioengineer) [inline R/zzz.R:1]: Interesting. I've not seen folks moving dependencies away from the functions they are called in. To me that has the potential of accidentally removing a dependency (import) when you think you no longer need it and cluttering up another location 🤔

Comment (elimillera) [reply R/zzz.R:1]: Here it is in [waldo ](https://github.com/r-lib/waldo/blob/master/R/waldo-package.R)and [dplyr](https://github.com/tidyverse/dplyr/blob/master/R/dplyr.r).  I think it mainly helps with redundancy and reducing namespace conflicts within the package itself. 

I agree there are pros and cons and I can switch it back if people disagree with this approach.

Comment (thebioengineer) [reply R/zzz.R:1]: in dplyr `@importFrom` is littered across the R code. I can see value in cases that there have been namespace conflicts identified calling the particulars (a la dplyrs issue with `vctr::data_frame()` vs `tibble::data_frame()`), and we resolve as such. but personally I think it is much clearer to list the imports for your function where they are needed.

definitely interested to hear @mariev 's interpretation

**Review by thebioengineer — APPROVED** [review]

File: R/zzz.R (hunk 1 above)
Comment (mariev) [reply R/zzz.R:1]: I think both paradigms are technically legitimate, but keeping `@importFrom ` with function level documentation is a better fit for review and long-term maintenance of a package, for the reasons that @thebioengineer stated initially. 

Definitely don't use a mix of the two in a single package!

Comment (elimillera) [reply R/zzz.R]: Works for me. importFrom has been moved back to the function documentation

**Review by thebioengineer — APPROVED** [review]
Thanks for making the updates @elimillera

**Review by mariev — CHANGES_REQUESTED** [review]
Looks good on visual inspection, but I am seeing errors when running devtools::check. Tried pulling in the `develop` branch, but that doesn't seem to resolve 🤷‍♀️ 

any thoughts on what is causing the test case failures? they seem to run via devtoools::test

**Comment by elimillera** [comment]
@mariev I think I remember resolving these earlier. I'll blame this one on git. I've turned on the R-CMD-Check for develop in the meantime. Good catch!

**Review by mariev — APPROVED** [review]
thanks for taking a second look 👍

**Comment by mariev** [comment]
@elimillera can we make a habit of deleting branches after they have been merged in? keeps things tidy

**Pull request merged**